modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -24,82 +24,80 @@ This page describes the options on the *Deploy Application* panel that is identi
 
 === From Anypoint Platform
 
-To deploy an application:
-
 . Sign in to the https://anypoint.mulesoft.com[Anypoint Platform] or click *Sign up* on the sign on page.
-. Hover your mouse over the *Runtime Manager* icon and click on it, or click *Open*
+. Hover your mouse over the *Runtime Manager* icon and click on it, or click *Open*.
 +
 image::deploying-to-cloudhub-5f383.png[]
 +
-. Click *Deploy application*:
+. In the *Applications* tab, click *Deploy application*.
 +
-image::deploying-to-cloudhub-3dcbb.png[]
+// image::deploying-to-cloudhub-3dcbb.png[]
 +
-.  The Deploy Application page opens:
+The Deploy Application page opens:
 +
 image::deploying-to-cloudhub-8524f.png[]
 
 
 === From SandBox
 
+If you created an application in a sandbox environment and tested it, when you are ready to deploy the application to production, you must first move the application.
 
-If you created an application in a sandbox environment and tested it, you would eventually want to migrate it to production.
-
-To do so:
+To move an app from a sandbox environment to production:
 
-. Logged into your Anypoint Platform account, go to your production environment
-. In the *Applications* tab, click the *Deploy Application* button
+. Log into your Anypoint Platform account, and go to your production environment.
+. In the *Applications* tab, click the *Deploy Application* button.
 +
-image::deploying-to-your-own-servers-d84f7.png[]
+// image::deploying-to-your-own-servers-d84f7.png[]
 
-. Then when picking the application file, instead of uploading a new file, click *Get from sandbox*.
+. When choosing the application file, click *Get from sandbox*.
 +
 image::deploying-to-cloudhub-d6f7a.png[]
 
-. You can then choose between your sandbox environments, and select an application to deploy in production. Select an application and click *Apply*
+. In the *Get from sandbox* dialog window, select your sandbox environment, then select an application to deploy in production, and click *Apply*.
```

Comment:
delete "window" after dialog?

Reply:
I do that out of habit. Last editor always made me add "box" or "window." But I prefer just saying dialog.